README.md

```diff
@@ -244,3 +271,63 @@ const wrapper = mount(
 #### Updating CRA
 
 When updating Create React App it's important to take a look at the `optimization.minimizer` values in the webpack config and then update the config in `craco.config.js`. After copying over any updates be sure to re-introduce the `terserOptions.mangle.reserved` key and values in the newly updated config.
+
+## Releasing the Dashboard
+
+To complete QA and release the Dashboard will take aproximately 2h of total time, 1h of which is an automated process at the end.
+
+- [Release prep](#release-prep)
+- [Generate a release tarball for Juju](#generate-a-release-tarball-for-juju)
+- [Release to jaas.ai](#release-to-jaas.ai)
+- [Release to Juju Simplestreams](#release-to-juju-simplestreams)
+- [Create an announcement on the Juju Discourse](#create-an-announcement-on-the-juju-discourse)
+
+### Release prep
+
+To prepare to release:
+
+- Clone a new copy of the repository to remove the chance of any uncommitted artifacts ending up in the release.
+- Thoroughly [QA the Dashboard](#qa'ing-the-dashboard) following the [Release tests](#release-tests)
+- Update the changelog by adding the important commits from the last tag.
+
+```
+  git log `git describe --tags --abbrev=0`..HEAD --format='- %b' --merges | sed '/^- $/d'
+```
+
+- Bump the version of the `package.json` file to the next appropriate semver version.
+- Commit the changes
+- Tag that commit with the appropriate version number.
+
+### Generate a release tarball for Juju
+
+The version of the Dashboard that's installed with every new Juju bootstrap is generated using:
+
+```
+./run exec yarn run generate-release-tarball
+```
+
+- Take this tarball and create a new release on GitHub under the appropriate version tag.
+- Add the notes from the changelog to the release notes on GitHub.
+
+### Release to jaas.ai
+
+The release to jaas.ai is in two steps. New releases of the jaas.ai website will pull in the
+latest release of the Dashboard for JAAS.
+
+- Visit the CI instance and release a new version of jaas.ai to staging.
+- Validate that the website works as expected and that the [release tests](#release-tests) are pass successfully.
+- Visit the CI instance and perform the release to jaas.ai production.
+- Validate that the website works as expected and that the [release tests](#release-tests) are pass successfully.
+
+### Release to Juju Simplestreams
+
+When Juju bootstraps a new instance it fetches the appropriate content for the infrastructure it's deployed on from a system called Simplestreams.
+
+To release a tarball of the Dashboard to Simplestreams:
+
+- Clone the release script [Shipit](https://github.com/CanonicalLtd/shipit).
+- Follow the instructions in that projects `README.md` to perform the release. This will take aproximately an hour but it's automated. You'll want to keep an eye on it as it's doing its thing in the event it needs input.
```

Comment:
Do you need access our permission to sign the gpg key and push?

Reply:
With the latest update to `shipit` it no longer requires that, but you have to be in the appropriate groups on launchpad.